pennylane/debugging/debugger.py

```diff
@@ -132,33 +132,198 @@ def pldb_device_manager(device):
 
 
 def breakpoint():
-    """Launch the custom PennyLane debugger."""
+    """A function which freezes execution and launches the PennyLane debugger (PLDB).
+
+    This function marks a location in a quantum circuit (QNode). When it is encountered during
+    execution of the quantum circuit, an interactive debugging prompt is launched to step
+    throught the circuit execution using Pdb like commands (:code:`list`, :code:`next`,
+    :code:`continue`, :code:`quit`).
+
+    **Example**
+
+    Consider the following python script containing the quantum circuit with breakpoints.
+
+    .. code-block:: python3
+
+        dev = qml.device("default.qubit", wires=2)
```

Comment:
```suggestion
        dev = qml.device("default.qubit")
```
It's fine to include, but we can skip the `wires` argument since switching to the new device API.

Reply:
Yes for sure, this is because when we call `qml.debug_state()` on a circuit while in execution, we display all of the device wires. If we remove the wires argument, then it will only display those wires which the circuit has encountered so far in the execution.

Reply:
I can remove it from this example as its not required for it.